.github/workflows/choco-package.yml

```
@@ -1,37 +1,44 @@
 name: Remote Dispatch Action
 on: [repository_dispatch]
 jobs:
+  package-push-common:
+      runs-on: windows-latest
+      outputs:
+        version: ${{ steps.get_version.outputs.version }}
+      steps:
+          - name: Event Information
+            run: |
+              echo "Event '${{ github.event.action }}' payload '${{ github.event.client_payload.tag }}'"
+              echo "Windows gardenlogin binary hash '${{ github.event.client_payload.windows_sha }}'"
```

```suggestion
              echo "Windows binary hash '${{ github.event.client_payload.windows_sha }}'"
```
remove component specific name